Cheetah/c/_namemapper.c

```diff
@@ -255,7 +300,8 @@ static PyObject *namemapper_valueForName(PYARGS)
 
     createNameCopyAndChunks();
 
-    theValue = PyNamemapper_valueForName(obj, nameChunks, numChunks, executeCallables, useDottedNotation);
+    /* passing 0 for isFirst, VFN is only called like VFN(VFFSL(...), ...) */
```

Comment:
I don't think that's a valid assumption.
People can easily call VFN however they like. It's part of the documented interface.